uiowa/uiowa
# Badge visibility

This PR solves #6417. It hides event status if it is scheduled, makes it orange if cancelled, and green if rescheduled.

# How to test

1. Sync admissions and iswf.
2. go to the /events of both sites
3. edit some events to give them cancelled and rescheduled
4. ensure that the badges properly show as defined in the synopsis of this PR.
5. Also ensure that, if there is a location, the badge is above it.
